### PR TITLE
NCC Angelic Sleuth

### PR DESCRIPTION
**Pull request by TheBlackMarvel**
Angelic Sleuth

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Angelic Sleuth
+ManaCost:2 W
+Types:Creature Angel Advisor
+PT:2/3
+K:Flying
+
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Permanent.YouCtrl+Other+HasCounters | TriggerZones$ Battlefield | Execute$ TrigInvestigate | TriggerDescription$ Whenever another permanent you control leaves the battlefield, if it had counters on it, investigate.
+SVar:TrigInvestigate:DB$ Investigate
+
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt:9]: DeckHints:Ability$Counters

File: forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Angelic Sleuth
+ManaCost:2 W
+Types:Creature Angel Advisor
+PT:2/3
+K:Flying
+
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt:6]: Empty lines would need to be removed from the final script.

File: forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt (hunk 1 above)
Comment (TheBlackMarvel) [reply forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt]: Added, thank you.

File: forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt (hunk 2 above)
Comment (TheBlackMarvel) [reply forge-gui/res/cardsfolder/upcoming/angelic_sleuth.txt]: Done, will remember to do so in future too!

**Pull request merged**